src/promptflow-tracing/promptflow/tracing/_experimental/utils.py

```diff
@@ -0,0 +1,12 @@
+from typing import Dict
+
+from opentelemetry import trace
+
+from promptflow.tracing._trace import serialize_attribute
+
+
+def enrich_prompt_template(template: str, variables: Dict[str, object]):
```

Comment:
I think the original function could be still in `_trace`, then we simply `from .._trace import enrich_prompt_template`, and we can share the logic in _trace

Reply:
refine the logic, share the logic in _trace